packages/config/src/bin/dev-script.ts

```diff
@@ -28,11 +28,18 @@ async function setEnv(cwd) {
     });
 }
 
-async function devScript(argv, cwd) {
+async function devScript(
+  argv: {
+    webpackConfig?: string;
+    clouddotEnv?: string;
+    uiEnv?: string;
+  },
+  cwd: string
+) {
   try {
     validateFECConfig(cwd);
 
-    const fecConfig = require(process.env.FEC_CONFIG_PATH);
+    const fecConfig = require(process.env.FEC_CONFIG_PATH!);
```

Comment:
I'm not sure if the trailing `!` is really what we want to do in these various locations where `process.env.FEC_CONFIG_PATH` is accessed... Are we actually guaranteed the env var will be set and not null/undefined?

Reply:
The `FEC_CONFIG_PATH`  is validated in the root of the `fec.js` binary. If it does not exist or is not valid, these scripts will not even be triggered. 

You can check the `validateFECConfig` function in the `common.ts` file.